README.md

```diff
@@ -55,8 +55,62 @@ To create custom special processes, add them into the `switch` statement in `Cus
 
 You can add custom item or move effects in `CustomApplyItemEffect` and `CustomApplyMoveEffect`.
 
+Please note that custom move effects are currently *not* handled by the *Metronome* move.
+
 #### Compatiblity with existing patches
-This project aims to keep compatibility with existing patches for move, item and special process effects to some degree. Special process effects using the `ExtractSpCode` patch can be reused without problems if they were imported with an ID lower than 100. Compatiblity with the `ExtractMoveCode` has not been thoroughly tested yet and might potentially cause issues with the *Metronome* move.
+
+**Note: ROMs patched with c-of-time currently experience crashes with the `ExtractSpCode` and `ExtractMoveCode` patch.**
+
+You can work around the crash by removing `.open "overlay29.bin", overlay29_start` and all following lines in `patches/internal.asm`.
+However, doing so will cause custom effects written in C to have no effect.
+
+Please reach out to us [on Discord](https://discord.gg/skytemple) for potential workarounds if you need to use the aforementioned patches while also adding effects via c-of-time.
+
+### Custom script engine instructions
+
+Custom script engine instructions are a more complex, but more powerful alternative to special processes.
+
+Advantages over special processes include:
+- No frame delay (especially beneficial when building complex minigames or other real-time interactions)
+- Custom instructions can be used inside targeted routines, while `ProcessSpecial` doesn't work
+- Cleaner script code overall without resorting to macros
+
+Disadvantages:
+- No built-in support in SkyTemple (workaround provided below)
+- No support for return values at the moment (the variable $EVENT_LOCAL is used as a "return register" by convention)
+
+Custom instructions are disabled by default. Follow these steps to enable support for custom instructions in c-of-time:
+1. Remove `#if 0` and `#endif` in `src/ground_instructions.c`
```

Comment:
It might be marginally easier to define a single C macro somewhere (`cot/custom_instructions.h` maybe?) and condition on that? Then you only need to change one line for all the C stuff.

Reply:
Good idea, that's a bit more convenient.